domain/src/main/java/org/oppia/android/domain/exploration/ExplorationProgressController.kt

```diff
@@ -129,6 +129,8 @@ class ExplorationProgressController @Inject constructor(
 
   private var mostRecentCommandQueue: SendChannel<ControllerMessage<*>>? = null
 
+  // the amount of time to wait before the continue interaction button is animated in milliseconds
+  private val continueButtonAnimationDelay: Long = 45000L
```

Comment:
```suggestion
  private val continueButtonAnimationDelayMs: Long = 45000L
```

Reply:
Also, I suggest using TimeUnit to calculate the value (see my reference module links above for a sample).

Reply:
done